CHANGELOG.rst

```diff
@@ -14,6 +14,24 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[1.2.9] - 2020-06-30
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Don't return dates for enrollments originally created too close to the
+course end to allow for finishing the course in time.
+
+[1.2.8] - 2020-06-17
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Don't return dates for enrollments created after course end
```

Comment:
I think these should be updated to say that it won't return assignment due dates (or perhaps just due dates). Only saying dates makes it seem like no dates are returned, but it still has start and end dates being returned.

Reply:
Aight.